outputs.tf

```diff
@@ -0,0 +1,6 @@
+output "webapp_url" {
+    value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+output "webhook" {
```

Comment:
We should [mark this value as sensitive](https://www.terraform.io/docs/language/values/outputs.html#sensitive-suppressing-values-in-cli-output) so that it doesn't get printed out in the logs since it contains an app service password